string name

geometry_msgs/WrenchStamped contact_force

geometry_msgs/Vector3Stamped surface_normal

geometry_msgs/Vector3Stamped contact_position

bool is_contact
